tests/engineer/test_nowcast.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+# import pickle
+import numpy as np
+# import xarray as xr
+
+from src.engineer import NowcastEngineer
+
+from ..utils import _make_dataset
+from .test_engineer import TestEngineer
+
+
+class TestNowcastEngineer(TestEngineer):
```

Comment:
Since the `stratify_xy` functions differ between the engineers, could we add tests to make sure they work?

Reply:
(Noticing now you have already mentioned this)

Reply:
todo thanks!

Reply:
done thanks mate